Tests/Algorithm/Framework/Alphas/CommonAlphaModelTests.cs

```diff
@@ -41,6 +42,9 @@ public abstract class CommonAlphaModelTests
         [TestFixtureSetUp]
         public void Initialize()
         {
+            var pythonPath = new DirectoryInfo("../../../Algorithm.Framework/Alphas");
+            Environment.SetEnvironmentVariable("PYTHONPATH", pythonPath.FullName);
+
```

Comment:
I think this should already be covered by the new `PythonSetup` class, which already sets the path for all models

Reply:
Reverted the changes in that file and the issue is still gone.